### PR TITLE
Use mr.scripty to define lines for openerp recipe addons based on sec…

### PR DESCRIPTION
**Pull request by yvaucher**
…tion [addons_versions] this to avoid having to write for each line 3 times the repository name

An exemple of use in camptocamp/camptocamp_openerp#43

This creates a variable `${scripty:github_addon_list}` containing a list of addons created with the list in addons_version having a value which can be splitted by (owner, tag). For example: carrier-delivery=oca 8.0 

It is backward compatible adding or not the `${scripty:github_addon_list}` variable in addons. As it lists only the addons version made of 2 elements.

This in order to ease addition of repositories and reduce size of common.cfg.

This also avoid error such as forgetting to put the version in `addons_version`

TL; DR
---------

We had:

    [openerp]
    addons=
    ...
        git git@github.com:OCA/webkit-tools.git parts/webkit-tools ${addons_version:webkit-tools}
    ...
    [addons_version]
        webkit-tools=8.0

This make it:

    [openerp]
    addons =
    ...
        ${scripty:github_addon_list}
    ...
    [addons_version]
        webkit-tools=oca 8.0

### CODE REVIEW

**Comment by yvaucher** [comment]
Applied in https://github.com/camptocamp/camptocamp_openerp/pull/43

**Comment by mdietrichc2c** [comment]
:+1:

**Comment by vrenaville** [comment]
:+1:
